app/src/main/java/org/coepi/android/ui/symptoms/SymptomsViewModel.kt

```diff
@@ -60,8 +73,35 @@ class SymptomsViewModel(private val symptomRepo: SymptomRepo) : ViewModel() {
     fun onChecked(symptom: SymptomViewData) {
         checkedSymptomTrigger.onNext(symptom)
     }
+    fun doSubmitSymptms(){
+        log.i("About to send sypmtom:");
+        if( realmprovider!= null ){
+            var keys: String = "will be put by doPostSymptom using last 3 keys generated";
+            var report = "COVID 19 confirmed"
+            var id = 1;
+            var symptomReport = SymptomReport(id, report,keys);
+            var retrofit = Retrofit.Builder()
+                .addCallAdapterFactory(RxJava2CallAdapterFactory.create())
+                .addConverterFactory(GsonConverterFactory.create())
+                .baseUrl("https://coepi.wolk.com:8080")
+                .build()
```

Comment:
Don't call anything api related directly from view models. You have to work here only with repositories. The repository calls DAO (Realm) and API.